OMFAPI/Program.cs

```diff
@@ -61,8 +59,19 @@ public static bool RunMain(bool test = false, Dictionary<string, dynamic> lastSe
                 foreach (Endpoint endpoint in endpoints)
                 {
                     if ((endpoint.VerifySSL is bool boolean) && boolean == false)
+                    {
                         Console.WriteLine("You are not verifying the certificate of the end point.  This is not advised for any system as there are security issues with doing this.");
```

Comment:
should we consider making this more clear what endpoint we aren't doing cert checks on

Reply:
agreed, what do you think of the new messaging?